.travis.yml

```diff
@@ -61,6 +62,11 @@ deploy:
           condition: $TRAVIS_BRANCH = "master" || ! -z "${TRAVIS_TAG}"  # master branch or tags
           repo: GeoscienceAustralia/fc
           python: "3.6"
+ -  provider: script
+    script: s3pypi --bucket datacube-core-deployment
+    on:
+        all_branches: true  # Let the condition below decide if the branch is to be deployed
+        condition: $TRAVIS_BRANCH = "master" || ! -z "${TRAVIS_TAG}"  # develop branch or tags
 
```

Comment:
Add `repo: GeoscienceAustralia/fc` so it doesn't attempt it on forks?

Reply:
Hmm, yeah, that's a good idea!
I removed it without thinking through why it was there...